test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/ActionApplicationModelConventionTest.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Reflection;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.ApplicationModels
+{
+    public class ActionApplicationModelConventionTest
+    {
+        [Fact]
+        public void DefaultActionModelConvention_AppliesToAllActionsInApp()
+        {
+            // Arrange
+            var app = new ApplicationModel();
+            app.Controllers.Add(new ControllerModel(typeof(HelloController).GetTypeInfo(), new List<object>()));
+            app.Controllers.Add(new ControllerModel(typeof(WorldController).GetTypeInfo(), new List<object>()));
+            var defaultConvention = new ApplicationModelConventionExtensions.ActionApplicationModelConvention(
```

Comment:
IMO, just think about using the extension method here and keep the class private. You don't have to try to hard to test this without making the classes internal